platformLibs/src/platform/windows/opengl32.def

```diff
@@ -0,0 +1,5 @@
+depends = windows posix
+package = platform.opengl32
+headers = wtypes.h minwindef.h wtypes.h GL/gl.h GL/wglext.h
```

Comment:
Why `wtypes.h` is specified twice?

Reply:
I didn't noticed. I think I copied the gdiplus.def one:
https://github.com/JetBrains/kotlin-native/blob/b34bfabe468c59fb43c6361ae67b8eb09a35ac21/platformLibs/src/platform/windows/gdiplus.def#L3
It is repeated there too